arex-web-core/src/main/java/com/arextest/web/core/business/config/replay/AbstractComparisonConfigurableHandler.java

```diff
@@ -101,11 +117,35 @@ public boolean insertList(List<T> configurationList) {
                     item.setExpirationDate(new Date());
                 }
             }).collect(Collectors.toList());
-
+        addDependencyId(configurations);
         return repositoryProvider.insertList(configurations);
     }
 
     public boolean removeByAppId(String appId) {
         return repositoryProvider.listBy(appId).isEmpty() || repositoryProvider.removeByAppId(appId);
     }
+
+    private void addDependencyId(List<T> comparisonDetails) {
```

Comment:
Why not batch query and batch upsert?

Reply:
Primary slave synchronization issue for reading and writing and need return id to do further